pset-iterators.js

```diff
@@ -93,6 +106,9 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+//  grab all presidential party names from presidents object and print to console
+const presidentialParties = presidents.map(presidentObj => presidentObj.party)
+console.log(presidentialParties)
```

Comment:
@sad3xoxo Good Job here using the map to return only the president's party names.